rpcserver.go

```diff
@@ -1434,7 +1434,8 @@ func (r *rpcServer) PendingChannels(ctx context.Context,
 
 		// If the channel was force closed, then we'll need to query
 		// the utxoNursery for additional information.
-		case channeldb.ForceClose:
+		// TODO(halseth): distinguish remote and local case?
+		case channeldb.LocalForceClose, channeldb.RemoteForceClose:
```

Comment:
Could do so by extending the RPC layer, possibly left to a follow up PR driven by a new issue.